### DISCUSSION TITLE
CMake Error at CMakeLists.txt:287 (message):

### DISCUSSION THREAD

**Discussion by wyu11071341** [micropython/micropython#12109] [RP2040 / Pico]
CMake Warning at /home/x/micropython/lib/pico-sdk/src/rp2_common/pico_cyw43_driver/CMakeLists.txt:15 (message):
  PICO_CYW43_DRIVER_PATH specified but content not present.

Found User C Module(s):
CMake Error at CMakeLists.txt:287 (message):
   cyw43-driver not initialized.
   Run 'make BOARD=PICO_W submodules'

？？？

**Comment by dlech** [comment]
Did you run `make BOARD=PICO_W submodules` as suggested in the error message?

**Reply by wyu11071341** [reply]
Yes, I have implemented these suggestions, but the same error message is still displayed.Do you have any other suggestions?

**Reply by jimmo** [reply]
Can you please share the output of `make BOARD=PICO_W submodules` ?

**Reply by wyu11071341** [reply]
The problem with PICO_CYW43_DRIVER_PATH seems to have been solved, but now a new problem is encountered:

CMake Error at CMakeLists.txt: 463 (pico_find_compiler)
Unknown CMake command "pico_find_compiler"

Any suggestions on how to solve it?

**Reply by jimmo** [reply]
you likely need to install arm-none-eabi-gcc
